gcsio/src/main/java/com/google/cloud/hadoop/gcsio/BatchHelper.java

```diff
@@ -71,15 +71,15 @@ public class BatchHelper {
    */
   public static class Factory {
     public BatchHelper newBatchHelper(
-        HttpRequestInitializer requestInitializer, Storage gcs, long maxRequestsPerBatch) {
+        HttpRequestInitializer requestInitializer, Storage gcs, int maxRequestsPerBatch) {
       return new BatchHelper(requestInitializer, gcs, maxRequestsPerBatch, /* numThreads= */ 0);
     }
 
     BatchHelper newBatchHelper(
         HttpRequestInitializer requestInitializer,
         Storage gcs,
-        long maxRequestsPerBatch,
-        long totalRequests,
+        int maxRequestsPerBatch,
+        int totalRequests,
```

Comment:
why changing this?

Reply:
For consistency - all constructor parameters are `int` now, and we do not need them to be `long`.